molgenis-data-import/src/main/java/org/molgenis/data/export/mapper/AttributeMapper.java

```diff
@@ -0,0 +1,165 @@
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_ENUM_OPTIONS, AttributeMetadata.ENUM_OPTIONS);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_PART_OF_ATTRIBUTE, AttributeMetadata.PARENT);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_RANGE_MAX, AttributeMetadata.RANGE_MAX);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_RANGE_MIN, AttributeMetadata.RANGE_MIN);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_MAPPED_BY, AttributeMetadata.MAPPED_BY);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_EXPRESSION, AttributeMetadata.EXPRESSION);
+    ATTRIBUTE_ATTRS.put(
+        EMX_ATTRIBUTES_VALIDATION_EXPRESSION, AttributeMetadata.VALIDATION_EXPRESSION);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_DEFAULT_VALUE, AttributeMetadata.DEFAULT_VALUE);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_TAGS, AttributeMetadata.TAGS);
+  }
+
+  private AttributeMapper() {}
+
+  public static List<Object> map(Attribute attr) {
+    List<Object> row = Lists.newArrayList();
+    for (Entry<String, String> entry : ATTRIBUTE_ATTRS.entrySet()) {
+      switch (entry.getKey()) {
+        case EMX_ATTRIBUTES_ID_ATTRIBUTE:
+          row.add(getIdAttrValue(attr));
+          break;
+        case EMX_ATTRIBUTES_TAGS:
+          row.add(getTagsValue(attr));
+          break;
+        case EMX_ATTRIBUTES_ENTITY:
+          row.add(attr.getEntity() != null ? attr.getEntity().getId() : "");
+          break;
+        case EMX_ATTRIBUTES_REF_ENTITY:
+          row.add(attr.getRefEntity() != null ? attr.getRefEntity().getId() : "");
+          break;
+        case EMX_ATTRIBUTES_LOOKUP_ATTRIBUTE:
+          row.add(getLookupValue(attr));
+          break;
+        case EMX_ATTRIBUTES_NILLABLE:
+          row.add(getNullableValue(attr));
+          break;
+        case EMX_ATTRIBUTES_VISIBLE:
+          row.add(getVisibleValue(attr));
+          break;
+        case EMX_ATTRIBUTES_PART_OF_ATTRIBUTE:
+          row.add(getPartOfValue(attr));
+          break;
+        case EMX_ATTRIBUTES_MAPPED_BY:
+          row.add(getMappedByValue(attr));
+          break;
+        case EMX_ATTRIBUTES_ENUM_OPTIONS:
+          row.add(getEnumOptions(attr));
+          break;
+        default:
+          Object value = attr.get(entry.getValue());
+          row.add(value != null ? value.toString() : "");
+      }
+    }
+    return row;
+  }
+
+  private static String getMappedByValue(Attribute attr) {
+    Attribute mappedBy = attr.getMappedBy();
+    return mappedBy != null ? mappedBy.getName() : "";
+  }
+
+  private static String getEnumOptions(Attribute attr) {
+
+    return attr.getEnumOptions() != null ? StringUtils.join(attr.getEnumOptions(), ",") : "";
+  }
+
+  private static String getLookupValue(Attribute attr) {
+    Boolean lookup = attr.getLookupAttributeIndex() != null;
+    return lookup.toString();
+  }
+
+  private static Object getTagsValue(Attribute attr) {
+    return StringUtils.join(
+        StreamSupport.stream(attr.getTags().spliterator(), false)
+            .map(Tag::getId)
+            .collect(Collectors.toList()),
```

Comment:
Static import Collectors.toList:

```suggestion
            .collect(toList());
```